### PR TITLE
Add SingleCell support for SCP; cleaned up references

### PR DESCRIPTION
**Pull request by kreinold**
Specifics about the implications of the changes are [DSPDC-1039](https://broadinstitute.atlassian.net/browse/DSPDC-1039) 

Also includes [DSPDC-1023](https://broadinstitute.atlassian.net/browse/DSPDC-1023)

### CODE REVIEW

File: src/terra-core/TerraCoreDataModel.ttl [hunk 1]
```diff
@@ -968,6 +977,16 @@ TerraCore:hasAlpha2Code
   rdfs:label "hasAlpha2Code" ;
   rdfs:range xsd:string ;
 .
+TerraCore:hasAnatomicalRegion
+  a owl:DatatypeProperty ;
+  rdfs:comment "Many groups are convening to define Common Coordinate Frameworks for specific organs/species.  This will be a rapidly evolving area in the near future.  Thus allowing text or URI as we wait for the field to mature allows us to support what we hope will at a minimum be controlled vocabulary under discussion of these groups." ;
+  rdfs:domain TerraCore:BioSample ;
+  rdfs:label "hasAnatomicalRegion" ;
+  rdfs:range xsd:anyURI ;
```
Comment (snwessel) [inline src/terra-core/TerraCoreDataModel.ttl:985]: Out of curiosity, what is the difference between this `xsd:anyURI` range and the `dct:URI` range that's used on hasDevelopmentalStage?

Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:985]: They should map to one another.  xsd is from the namespace for XML Schema and dct is the Dublin Core Terms which is a foundational ontology.  Both are likely used by anyone who might use a data model and both are required by this data model.  On the other hand, we should be consistent.  Looks like I've used it inconsistently.  Would you write up a ticket to define the strategy and update all our data models.  We should not modify others.  Good catch!

Comment (snwessel) [reply src/terra-core/TerraCoreDataModel.ttl:985]: Sounds good! I'll write up a ticket. And thanks for the explanation!

Comment (snwessel) [reply src/terra-core/TerraCoreDataModel.ttl:985]: Here's the ticket I made: https://broadinstitute.atlassian.net/browse/DSPDC-1147

And feel free to add anything I left out.

File: src/references/OBI/OBI_assaySubset.xml.rdf [hunk 2]
```diff
@@ -8407,8 +8364,976 @@
...
-        <IAO_0000412 rdf:resource="http://purl.obolibrary.org/obo/obi.owl"/>
+        <obo:IAO_0000111>chromosome conformation capture sequencing assay</obo:IAO_0000111>
+        <obo:IAO_0000412 rdf:resource="http://purl.obolibrary.org/obo/obi.owl"/>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OBI_0002621 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0002621">
```
Comment (quazi-broad) [inline src/references/OBI/OBI_assaySubset.xml.rdf:8375]: How do we know which assays to include in this list? Or is it an all encompassing list of assays and these have just been added to the ontology recently?

**Comment by kreinold** [comment]
I've taken all the assays from OBI.  We will reevaluate from time-to-time.

*Kathy Reinold*
*Principal Data Modeler, Data Sciences Platform *
The Broad Institute of Harvard and MIT
105 Broadway - 359M
Cambridge, MA 02142

**Pull request merged**